src/fileio/fileio_party.cpp

```diff
@@ -264,7 +264,10 @@ bool load_party_v1(fs::path file_to_load, cUniverse& real_univ, bool town_restor
 					}
 				}
 			}
-		else univ.party.town_num = 200;
+		else {
+			univ.party.town_num = 200;
+			univ.town.difficulty = 0;
```

Comment:
I decided to delete this. It looked like it was just causing confusion, and we should be using `cTown::difficulty` (`univ.town->difficulty`) instead.

Reply:
Yes, this seems better.